staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher_test.go

```diff
@@ -23,15 +23,11 @@ import (
 	"testing"
 	"time"
 
-	storagetesting "k8s.io/apiserver/pkg/storage/testing"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apimachinery/pkg/watch"
-	"k8s.io/apiserver/pkg/apis/example"
 	"k8s.io/apiserver/pkg/storage"
 	"k8s.io/apiserver/pkg/storage/etcd3/testserver"
+	storagetesting "k8s.io/apiserver/pkg/storage/testing"
+
```

Comment:
```suggestion
```

/hold 
for go format ci verify failure.

Reply:
Done